config/locales/en.yml

```diff
@@ -2450,17 +2450,17 @@ en:
       local_chapters:
         title: Local Chapters
         about_text: |
-          Local Chapters are country-level or region-level groups that have taken the formal step of
-          establishing not-for-profit legal entities. They represent the area's map and mappers when
+          Local chapters are national or regional groups that have taken the formal step of
+          establishing nonprofit legal entities. They represent the area's map and mappers when
```

Comment:
At least one of the local chapters, OpenStreetMap U.S., is a nonprofit organization. It is _not_ a not-for-profit organization. The two terms are [legally quite distinct](https://www.indeed.com/career-advice/career-development/non-profit-vs-not-for-profit-vs-for-profit), and mixing them up could dissuade some people from joining or donating to OSMUS.

Reply:
Your change seems to claim that all local chapters are nonprofit legal entities. Even if this were true at the moment - which I'm not sure about -, it is not required that a new local chapter must be nonprofit. The [LC template agreement](https://osmfoundation.org/wiki/Local_Chapters/Template_agreement) only says they must be not-for-profit. Your change is therefore misleading. It may well be that OSM US are exceeding the requirements for a local chapter in this regard but I think that the OSM US website would be the correct place to mention that.

Reply:
Well it already says that unless you're going to argue that `not-for-profit` and `nonprofit` somehow have different meanings?

Reply:
As I understand the U.S. situation, nonprofit and not-for-profit are both types of tax-exempt; nonprofit isn’t not-for-profit-deluxe. But I guess we don’t want to detract from the explanation that follows – the user will care more about the service provided than the legal status in this context. Should we avoid either term and just say it’s a legal entity, unqualified? If we need to emphasize that these entities are noncommercial, we could use that term.